app/controllers/planning_applications/consultee/emails_controller.rb

```diff
@@ -81,6 +82,14 @@ def redirect_to_application_page
       def public_or_preapp?
         @planning_application.make_public? || @planning_application.pre_application?
       end
+
+      def ensure_consultation_required
+        return unless @planning_application.pre_application?
```

Comment:
See above: if we make this line return true, then we reduce some of the duplication elsewhere:

```suggestion
        return true unless @planning_application.pre_application?
```